hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/TestAccountConfiguration.java

```diff
@@ -382,6 +382,7 @@ public void testConfigPropNotFound() throws Throwable {
 
     for (String key : CONFIG_KEYS) {
       setAuthConfig(abfsConf, true, AuthType.OAuth);
+      abfsConf.unset(key);
```

Comment:
Can you please share in description why we are unsetting it. Does this get ignored somehow in current tests.

Reply:
This is a fix for this test failure :- TestAccountConfiguration.testConfigPropNotFound:386->testMissingConfigKey:399 Expected a org.apache.hadoop.fs.azurebfs.contracts.exceptions.TokenAccessProviderException to be thrown, but got the result: : "org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider"